components/brave_new_tab_ui/reducers/cryptoDotCom_reducer.ts

```diff
@@ -31,13 +34,32 @@ const cryptoDotComReducer: Reducer<NewTab.State | undefined> = (state: NewTab.St
   switch (action.type) {
     case types.ON_BTC_PRICE_OPT_IN:
       state = { ...state }
-      state.cryptoDotComState.optInBTCPrice = true
+      state.cryptoDotComState = {
+        ...state.cryptoDotComState,
+        optInBTCPrice: true,
+        fetchStatus: 'pending'
+      }
+      performSideEffect(async function () {
+        chrome.cryptoDotCom.onInteraction()
+      })
       break
 
-    case types.MARKET_DATA_UPDATED:
+    case types.MARKETS_REQUESTED:
+      state.cryptoDotComState = {
```

Comment:
Missing `state = {...state}`. Without that, `oldState === newState` and no re-render will occur.

Reply:
Ahh right. Should be fixed now. Thanks for catching!